ocaml/doc/README.md

```diff
@@ -1,13 +1,14 @@
 # A note on generating locally the API reference
 
-Run `make doc`. The output is in `_build/ocaml/doc/markdown`.
+Run `make doc`. The output is in `_build/ocaml/doc`.
```

Comment:
I think this section could be made clearer.

---

* Running `make doc` creates documentation in HTML format in `_build/ocaml/doc`. This requires `graphiv` to be installed for generating images from `*.dot` files

* To generate documentation in PDF format, run `sh doc-convert.sh` in `_build/ocaml/doc`. This requires `pandoc` and `pdflatex` being installed.

* To generated documentation in Docbook format, run `sh doc-convert.sh --docook` in `_build/ocaml/doc`. This requires `pandoc` being installed. 

---

Why not provide makefile targets for these? 

* `make-doc`
* `make-doc-html`
* `make-doc-pdf`
* `make-doc-docbook`